src/mesh/python/mesh_types.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# -------------------------------------------*-python-*------------------------------------------- #
```

Comment:
Please add standard copyright block. to both python files.

Reply:
Ah - thank you for catching this - will do.

Reply:
Just added these fields - thank you!